Gemfile

```diff
@@ -1,31 +1,32 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.0.2"
+gem 'rubocop', '>= 1.0', '< 2.0'
+
+ruby '3.0.2'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.5", ">= 7.0.5.1"
 
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
-gem "sprockets-rails"
+gem 'sprockets-rails'
 
 # Use postgresql as the database for Active Record
-gem "pg", "~> 1.1"
+gem 'pg', '~> 1.1'
```

Comment:
- [OPTIONAL] I don't seem to see rails gem in your Gemfile and this is what I got while trying to setup your app on my local machine.

### Error
![your-error](https://github.com/FrankMugagga/blog_app/assets/26597432/eda9cdbc-0139-4e4b-b118-35462e4b0fa6)